### PR TITLE
Cumulus 1619 add information to logs

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** Adds three new optional keys to the Logger class

Addresses [CUMULUS-1619: Update Logs with better metadata](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1619)

## Changes

* Adds three new optional keys to the Logger class
  - `granules`, `parentArn`, and `stackName`

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/common/log.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ const isString = require('lodash.isstring');
 function logger() {
   return new Logger({
     executions: process.env.EXECUTIONS,
+    granules: process.env.GRANULES,
```
Comment (jennyhliu) [inline packages/common/log.js]: Is there a length limit on process.env.GRANULES?  Would it be an issue if we have a long list of granules?

Comment (yjpa7145) [inline packages/common/log.js]: Where are these environment variables being set?

Comment (flamingbear) [reply packages/common/log.js]: they're being set in the message adapter client js . https://github.com/nasa/cumulus-message-adapter-js/pull/28

It's optional, fo if it's not set, it just doesn't show up.

Comment (flamingbear) [reply packages/common/log.js]: Good point, I don't know if there's a limit to the size of an env var.  google says :  https://unix.stackexchange.com/questions/336934/raise-128kib-limit-on-environment-variables-in-linux
so 131072 characters.  Is that going to be a problem?

Comment (yjpa7145) [reply packages/common/log.js]: process.env.GRANULES is going to be a string, is there something that parses that into an array?

Comment (laurenfrederick) [reply packages/common/log.js]: @yjpa7145 I think env vars have to be a string? So maybe an array as JSON string that we can parse?

Comment (laurenfrederick) [reply packages/common/log.js]: Oh it already is a string, but I am going to change it to be just the ids.

Comment (flamingbear) [reply packages/common/log.js]: whatever we end up putting there will be parsed by the logstash on the metrics side during load to ES.

File: packages/common/aws.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const improveStackTrace = (fn) =>
       return await fn(...args);
     } catch (err) {
       setErrorStack(err, tracerError.stack);
-      err.message = `${err.message}; Function params: ${args}`;
+      err.message = `${err.message}; Function params: ${JSON.stringify(args, null, 2)}`;
```
Comment (flamingbear) [inline packages/common/aws.js]: Oh, I did this, but I'm not sure if we want to spread out the errors across all lines.  Maybe nuke the `, null, 2`?

Comment (kkelly51) [reply packages/common/aws.js]: Doesn't the null, 2 just put it in an easier to read format? I think that's fine

Comment (flamingbear) [reply packages/common/aws.js]: Yeah, I thought this was just leftover from my debugging, but Lauren added in on purpose so I'm ok with it.

File: packages/logger/README.md [hunk 3]
```diff
@@ -51,7 +52,7 @@ log.info('hello, world');
 
 ## API
 
-### `new Logger({ [executions], [granules], [parentArn], [sender], [stackName], [version] })`
+### `new Logger({ [executions], [granules], [parentArn], [sender], [stackName], [version], [asyncOperationId] })`
```
Comment (flamingbear) [inline packages/logger/README.md]: I was trying to keep these alphabetized for marc.

Comment (laurenfrederick) [reply packages/logger/README.md]: oh my gosh. sorry didn't notice that. I will fix it!

**Pull request merged**